week5_R_draft.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
<ul><li>maybe explain what mar is doing</li><li>I think you need comments to go into more detail of what each argument is doing so it is clear what is going on. Explain what bar plot is and etc.</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>